### PR TITLE
[AMORO-2941] Complete translations for amoro dashboard

### LINKED ISSUE #2941: [Improvement]: ams dashboard fully support i18n
### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

so far,  the i18n support of ams dashboard is very limited: Menu items Only.

### How should we improve?

All views should support i18n.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by chouchouji**
## Why are the changes needed?

Close #2941 .

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by chouchouji** [comment]
@zhoujinsong  I create a **md** file about translations, please review and help me to complete it. Thanks!

[translations.md](https://github.com/user-attachments/files/15956279/translation.md)

**Comment by zhoujinsong** [comment]
I updated some transaction:

```markdown
| 英文 | 中文 |
| --- | --- |
|  `Filter partitions` | `过滤分区` |
|  `Nothing to show`   | `无内容可展示` |
|  `delete-files`      | `删除文件数量` |
|  `data-files`        | `数据文件数量` |
|  `total-files`       | `总文件数量` |
|  `pos-delete-records`       | `Position 删除文件数据行数` |
|  `eq-delete-records`       | `Equality 删除文件数据行数` |
|  `total-records`       | `总数据行数` |
|  `Table file count`       | `表文件总数` |
|  `Table record count`       | `表记录总数` |
|  `Tasks`       | `任务` |
|  `properties.{type}.memory`       | `` |
|  `Basic`       | `基本` |
|  `Metastore`       | `元数据中心` |
|  `Table Format`       | `表格式` |
|  `List`       | `列表` |
|  `No.`       | `编号` |
|  `Top 10`       | `前 10` |
|  `Other Properties`       | `其他属性` |
|  `Partiton Field`       | `分区字段` |
|  `Upgrade Hive Table`       | `升级 Hive 表` |
|  `Occupation`       | `占用率` |
|  `Records`       | `数据行数` |
|  `Scale-Out`       | `扩容` |
|  `Core`       | `核` |
|  `Resource Occupation`       | `资源占用率` |
|  `Operation Details`       | `操作详情` |
|  `Optimize Type`       | `优化类型` |
|  `Touch Time`       | `心跳时间` |
|  `Task ID`       | `任务 ID` |
|  `Partition`       | `分区` |
|  `Basic Information`       | `基本信息` |
|  `Filter {inputPh}`       | `过滤 {inputPh}` |
|  `Please input {inputPh}`       | `请输入 {inputPh}` |
|  `Please select {selectPh}`       | `请选择 {inputPh}` |
|  `Quota`       | `配额` |
|  `Metric Value`       | `指标值` |
|  `Metric`       | `指标` |
|  `Transform`       | `转换函数` |
|  `Table Properties`       | `表属性` |
|  `Properties`       | `属性` |
|  `Base Table Metrics`       | `Base 表指标` |
|  `Change Table Metrics`       | `Change 表指标` |
|  `Main Metrics`       | `主要指标` |
|  `Schema`       | `结构` |
|  `Partition Key`       | `分区值` |
|  `Implementation`       | `实现方式` |
|  `Table \| Tables`       | `表` |
|  `Database \| Databases`       | `库` |
|  `Order`       | `排序` |
|  `Token`       | `Token` |
```

**Comment by chouchouji** [comment]
> I updated some transaction:
> 
> ```
> | 英文 | 中文 |
> | --- | --- |
> |  `Filter partitions` | `过滤分区` |
> |  `Nothing to show`   | `无内容可展示` |
> |  `delete-files`      | `删除文件数量` |
> |  `data-files`        | `数据文件数量` |
> |  `total-files`       | `总文件数量` |
> |  `pos-delete-records`       | `Position 删除文件数据行数` |
> |  `eq-delete-records`       | `Equality 删除文件数据行数` |
> |  `total-records`       | `总数据行数` |
> |  `Table file count`       | `表文件总数` |
> |  `Table record count`       | `表记录总数` |
> |  `Tasks`       | `任务` |
> |  `properties.{type}.memory`       | `` |
> |  `Basic`       | `基本` |
> |  `Metastore`       | `元数据中心` |
> |  `Table Format`       | `表格式` |
> |  `List`       | `列表` |
> |  `No.`       | `编号` |
> |  `Top 10`       | `前 10` |
> |  `Other Properties`       | `其他属性` |
> |  `Partiton Field`       | `分区字段` |
> |  `Upgrade Hive Table`       | `升级 Hive 表` |
> |  `Occupation`       | `占用率` |
> |  `Records`       | `数据行数` |
> |  `Scale-Out`       | `扩容` |
> |  `Core`       | `核` |
> |  `Resource Occupation`       | `资源占用率` |
> |  `Operation Details`       | `操作详情` |
> |  `Optimize Type`       | `优化类型` |
> |  `Touch Time`       | `心跳时间` |
> |  `Task ID`       | `任务 ID` |
> |  `Partition`       | `分区` |
> |  `Basic Information`       | `基本信息` |
> |  `Filter {inputPh}`       | `过滤 {inputPh}` |
> |  `Please input {inputPh}`       | `请输入 {inputPh}` |
> |  `Please select {selectPh}`       | `请选择 {inputPh}` |
> |  `Quota`       | `配额` |
> |  `Metric Value`       | `指标值` |
> |  `Metric`       | `指标` |
> |  `Transform`       | `转换函数` |
> |  `Table Properties`       | `表属性` |
> |  `Properties`       | `属性` |
> |  `Base Table Metrics`       | `Base 表指标` |
> |  `Change Table Metrics`       | `Change 表指标` |
> |  `Main Metrics`       | `主要指标` |
> |  `Schema`       | `结构` |
> |  `Partition Key`       | `分区值` |
> |  `Implementation`       | `实现方式` |
> |  `Table \| Tables`       | `表` |
> |  `Database \| Databases`       | `库` |
> |  `Order`       | `排序` |
> |  `Token`       | `Token` |
> ```

Thanks!

**Review by zhangmo8 — APPROVED** [review]
LGTM

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 1]
```diff
@@ -18,8 +18,188 @@
...
+  commitTime: '提交时间',
+  commitId: '提交 ID',
+  path: '路径',
+  snapshotId: '快照 ID',
+  fileType: '文件类型',
+  fileContent: '文件内容',
+  startTime: '开始时间',
+  finishTime: '完成时间',
+  touchTime: '心跳时间',
+  duration: '期间',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:87]: 耗时

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 2]
```diff
@@ -18,8 +18,188 @@
...
+  optimizeType: '优化类型',
+  parallelism: '并行',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:89]: 并行度

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 3]
```diff
@@ -18,8 +18,188 @@
...
+  input: '输入',
+  output: '输出',
+  count: '总数',
+  time: '时间',
+  operation: '操作',
+  operationDetails: '操作详情',
+  copy: '复制',
+  optimizer: '优化器',
+  optimizerId: '优化器 Id',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:98]: 优化器 ID

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 4]
```diff
@@ -18,8 +18,188 @@
...
+  optimizingStatus: '优化状态',
+  optimzerGroup: '优化器组',
+  allGroups: '全部优化器',
+  retry: '重试',
+  costTime: '花费时长',
+  thread: '线程',
+  resourceOccupation: '资源占用率',
+  core: '核',
+  scaleOut: '扩容',
+  release: '发布',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:111]: 释放

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 5]
```diff
@@ -18,8 +18,188 @@
...
+  optimizeStatus: '优化状态',
+  fileSize: '文件大小',
+  records: '数据行数',
+  occupation: '占用率',
+  optimizerGroup: '优化器组',
+  optimizerGroups: '优化器组',
+  resourceAllocation: '资源分配',
+  resourceGroup: '资源组',
+  releaseOptModalTitle: '发布此优化器？',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:120]: 释放此优化器？

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 6]
```diff
@@ -18,8 +18,188 @@
...
+  cancelOptimizingProcessOptModalTitle: '取消该表的优化过程？',
+  welecomeTip: '欢迎来到 Amoro!',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:122]: 欢迎访问 Amoro

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 7]
```diff
@@ -18,8 +18,188 @@
...
+  signin: '登入',
+  username: '用户名',
+  password: '密码',
+  pause: '暂停',
+  run: '启动',
+  format: '格式化',
+  fullscreen: '全屏',
+  recovery: '恢复',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:130]: 还原

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 8]
```diff
@@ -18,8 +18,188 @@
...
+  result: '结果',
+  noResult: '无结果',
+  log: '日志',
+  ready: '准备中',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:134]: 就绪

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 9]
```diff
@@ -18,8 +18,188 @@
...
+  upgradeHiveTable: '升级 Hive 表',
+  ok: '确定',
+  partitonField: '分区字段',
+  otherProperties: '其他属性',
+  errorMessage: '错误信息',
+  copySuccess: '复制成功',
+  duplicateKey: '重复键',
+  lastUpgradingFailed: '上次升级失败',
+  data: '数据',
+  resourceCpu: 'Cpu 使用情况',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:154]: CPU 使用情况

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 10]
```diff
@@ -18,8 +18,188 @@
...
   terminal: '终端',
   settings: '设置',
+  optimizergroup: '优化器组',
+  systemSetting: '系统设置',
+  containerSetting: '容器设置',
+  backHome: '返回首页',
+  version: '版本',
+  userGuide: '用户指南',
+  logout: '登出',
+  logoutModalTitle: '要退出ams吗？',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:34]: 要退出AMS吗？

**Review by tcodehuber — COMMENTED** [review]
LGTM, I left some comments, PTAL.

**Comment by chouchouji** [comment]
> @chouchouji LGTM, I left some comments, PTAL.

Thanks! I will update it tonight.

**Comment by chouchouji** [comment]
> @chouchouji LGTM, I left some comments, PTAL.

Done

File: amoro-ams/amoro-ams-dashboard/src/language/zh.ts [hunk 11]
```diff
@@ -18,8 +18,188 @@
...
+  leavePageModalTitle: '离开当前页面？',
+  leavePageModalContent: '您所做的更改可能无法保存。',
+  leave: '离开',
+  invalidInput: '非法输入',
+  addgroup: '添加组',
+  editgroup: '编辑组',
+  cannotDeleteGroupModalTitle: "不能移除这个组",
+  cannotDeleteGroupModalContent: '这个组已被使用',
+  deleteGroupModalTitle: '移除这个组',
+  processId: '进程ID',
```
Comment (tcodehuber) [inline amoro-ams/amoro-ams-dashboard/src/language/zh.ts:188]: 进程 ID

**Review by tcodehuber — APPROVED** [review]
LGTM, thanks for your work.

**Comment by chouchouji** [comment]
> LGTM, thanks for your work.

You are welcome. I am glad to make amoro better!

**Comment by zhoujinsong** [comment]
Thanks for the work! @chouchouji

**Pull request merged**